aws/studio_container_setup.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>Remove the outputs with "Clear all outputs"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
ok updated, thanks!